proteus.go

```diff
@@ -30,6 +30,8 @@ func transformToProtobuf(packages []string, generate generator) error {
 	r.Resolve(pkgs)
 
 	t := protobuf.NewTransformer()
+	t.SetStructSet(createStructTypeSet(pkgs))
```

Comment:
Moving this to an issue. Requires refactoring of most of the `Transformer` related code and tests.